src/frontend/package.json

```diff
@@ -1,11 +1,29 @@
 {
-    "name": "inventreeui",
+    "name": "inventree",
```

Comment:
is it wise to name this like the project? I think it would be better to use the namespace that you secured and call it something similar to @inventree/lib